deployment/route96/templates/deployment.yaml

```
@@ -0,0 +1,91 @@
+      volumes:
+      - name: blossom-config
+        configMap:
+          name: blossom-config
+      - name: blossom-storage
+        persistentVolumeClaim:
+          claimName: blossom-pvc
+      - name: database-cacert
+        secret:
+          secretName: blossom-secret
+          items:
+            - key: database_cacert
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 10 but found 12

(indentation)